x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/form.tsx

```diff
@@ -419,6 +419,7 @@ export const EventFiltersForm: React.FC<ArtifactFormComponentProps & { allowSele
                 comments: exception?.comments ?? [],
                 os_types: exception?.os_types ?? [OperatingSystem.WINDOWS],
                 tags: exception?.tags ?? [],
+                meta: exception.meta,
```

Comment:
Is this change related to the bug fix or something else you found?

Reply:
It is related to the `Event Filters` component
The component was throwing the below issue, when we updated `useEffect` with the deps 
![image](https://github.com/elastic/kibana/assets/12671903/1ee91cf0-71ba-4abd-8a35-e25d768ce644)